manual/trees/index.md

```diff
@@ -59,10 +59,11 @@ Branches with more data per tree entry will fill more baskets than branches with
 Conversely, baskets can hold many tree entries if their branch stores only a few bytes per tree entry.
 This means that generally, all baskets - also of different branches - will contain data of different tree entry ranges.
 
-To allow more efficient pre-fetching and better chunking of tree data stored in ROOT files, TTree groups baskets into clusters, for a defined range of tree entry indices.
+To allow more efficient pre-fetching and better chunking of tree data stored in ROOT files, TTree groups baskets into _clusters_.
+A cluster contains all the data of a given event range.
```

Comment:
Is it ok to talk about `event range` when so far we've talked about `entries` ?

Reply:
Fixed thanks!